### ISSUE TITLE
Support Clojure 1.9+

### ISSUE THREAD

**Issue by jaidetree** [hashobject/perun#227]
Perun cannot run entirely in Clojure 1.9+. This is due to a dependency on sitemap with a ns form that doesn't conform to the updated spec. The fix for sitemap was merged into master but there are a few steps involved before it can be updated in perun.

- [X] Patch sitemap to pass Clojure 1.9+ validation
- [X] Merged sitemap patch into master
- [X] Release new version of sitemap
- [X] Update sitemap dependency in perun deps. (waiting for #224)
- [ ] Release new version of perun

**Comment by allentiak** [comment]
On Mon, 11 Feb 2019 at 21:09, Jay <notifications@github.com> wrote:

>
>    - Release new version of sitemap
>
> @podviaznikov Could you please do this?


>
>    - Update sitemap dependency in perun deps.
>
>
I can take care of this one, once the previous task is done.

>

**Comment by jaidetree** [comment]
What needs to be done to release the next version of sitemap? No rush or anything :smile:, just want to understand the status of that project in relation to perun.

**Comment by podviaznikov** [comment]
released sitemap `0.4.0`.
let me know if anything is needed there https://clojars.org/sitemap

**Comment by allentiak** [comment]
@eccentric-j 
Once closed, #224 will take care of the third item.
I added a fourth item to the list for the released of an updated perun afterwards.

**Comment by jaidetree** [comment]
Great, thanks Podviaznikov. Added a note in the code review on #224 to bump sitemap to 0.4.0.

**Comment by allentiak** [comment]
@eccentric-j #224 has been merged

**Comment by jaidetree** [comment]
@allentiak Great, thanks.

@podviaznikov @allentiak What's next to be able to release the next version of perun?

**Comment by allentiak** [comment]
Give me a few days and I'll get back to you on this.

**Comment by allentiak** [comment]
I have tested perun with Clojure 1.10.1, and it works fine. (I'm closing this.)

(The only thing left to do would be the release, but I think we should first discuss what we would like to see there.)
